src/Database/PostgreSQL/PQTypes/SQL/Builder.hs

```diff
@@ -664,16 +692,25 @@ sqlWhereIsNULL col = sqlWhere $ col <+> "IS NULL"
 sqlWhereIsNotNULL :: (MonadState v m, SqlWhere v) => SQL -> m ()
 sqlWhereIsNotNULL col = sqlWhere $ col <+> "IS NOT NULL"
 
+-- | Run monad that joins all conditions with 'AND' operator.
+--
+-- When no conditions are given, the result is 'TRUE'.
+sqlAll :: State SqlWhereAll () -> SQL
+sqlAll = toSQLCommand . (`execState` (SqlWhereAll []))
+
+-- | Run monad that joins all conditions with 'OR' operator.
+--
+-- When no conditions are given, the result is 'FALSE'.
+sqlAny :: State SqlWhereAny () -> SQL
```

Comment:
Perhaps we should deprecate `sqlWhereAny` and start using `sqlWhere . sqlAny $ do ...` instead?

Reply:
Depends if you'd like to remove it eventually. But if you want to deprecate `sqlWhereAny`, then I'd like to deprecate `sqlOR`, `sqlConcatAND` and `sqlConcatOR` as well.

Reply:
If it comes to kontrakcja, `sqlOR` is used 1 time, `sqlConcatAND` 5 times and `sqlConcatOR` 4 times, so it seems feasible. Let's do it a separate PR though.